auslib/db.py

```diff
@@ -319,6 +319,10 @@ def insert(self, changed_by=None, transaction=None, **columns):
 
            @rtype: sqlalchemy.engine.base.ResultProxy
         """
+        if dryrun:
```

Comment:
I'm not sure what you mean? My top comment was just saying that I copied this from #151, because its needed here.

I'm not sure I see the value in moving this into prepareInsert, but maybe I'm missing something?